src/coreclr/vm/amd64/asmhelpers.S

```diff
@@ -307,3 +307,44 @@ NESTED_ENTRY ProfileTailcallNaked, _TEXT, NoHandler
 
   ret
 NESTED_END ProfileTailcallNaked, _TEXT
+
+#ifdef TARGET_OSX
+# EXTERN_C void* GetThreadVarsSectionOffset()
```

Comment:
Nit: This returns an address, not an offset.

Reply:
If i understand the teminology correctly, it retrieves the offset and then we do the final address calculation in C++ side, likewise for osx/arm64, hence I kept it "Offset". Let me know if that is not correct.

Reply:
Offset is an integer that gets added to the base address.

The value that this method returns is not added to anything, so it is not returning an offset.

(This comment is more applicable to the arm64 version of the method. The disassembling on x64 makes things murky - I have commented on it separately.)

Reply:
Fixed.